packages/core/src/utils/Graph.ts

```diff
@@ -153,6 +153,30 @@ export default class Graph<I, L = undefined, E = undefined> {
     return xs.reverse();
   }
 
+  /**
+   * throws if the graph does not contain node `i`
+   */
+  children(i: I): I[] {
+    const vert = this.g.get(i);
+    if (vert) {
+      return vert.s.map(({ i }) => i);
+    } else {
+      throw Error("Node does not exist");
+    }
```

Comment:
Use `this.get` instead for better concision and error handling:

```suggestion
    return this.get(i).s.map(({ i }) => i);
```